flask_webapp/src/device_list.py

```diff
@@ -57,7 +58,7 @@ def scan(self):
         :return: list of device name strings.
         """
         self._scan_for_bt_ble_devices()
-        self._scan_for_bt_regular_devices()
+        #self._scan_for_bt_regular_devices()
```

Comment:
In the test code it always returns both ble & non-ble devices. Is there something in the test code breaking the new code (blocking it from including both device types)?